File: sdkwork-app-sdk-typescript/src/types/project-list-query-request.ts
export interface ProjectListQueryRequest {
  /** 关键词 */
  keyword?: string;
  /** 工作空间ID */
  workspaceId?: string;
  /** 项目类型 */
  type?: string;
  /** 状态 */
  status?: string;
  /** 排序字段 */
  sortField?: string;
  /** 排序方向 */
  sortDirection?: string;
  /** 页码 */
  page?: number;
  /** 每页大小 */
  size?: number;
}

File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/ProjectListQueryRequest.kt
package com.sdkwork.app

data class ProjectListQueryRequest(
    val keyword: String? = null,
    val workspaceId: String? = null,
    val type: String? = null,
    val status: String? = null,
    val sortField: String? = null,
    val sortDirection: String? = null,
    val page: Int? = null,
    val size: Int? = null
)
